mozilla/foundation.mozilla.org
# Added GA event tracking to a footer link

Related to #1700

QA instructions

1. make sure you have "Do Not Track" browser setting turned off
2. make sure you disable privacy related add-ons
3. open [GA Dashboard](https://analytics.google.com/analytics/web/#/realtime/rt-event/a87658599w138395144p152070124/filter.list=41==page%252520footer%252520cta;40==navigation;&metric.type=5//) in a new tab
4. visit the following test links and click on the "See more" link near footer section
 <img width="964" alt="screen shot 2018-08-13 at 3 08 12 pm" src="https://user-images.githubusercontent.com/2896608/44060815-bcb98ef8-9f0a-11e8-9d86-42cd49d73197.png">

- https://foundation-mofostaging-pr-1740.herokuapp.com/campaigns/indeed-grow-health/
- https://foundation-mofostaging-pr-1740.herokuapp.com/campaigns/them-stand-sit/
- https://foundation-mofostaging-pr-1740.herokuapp.com/campaigns/interest-find-near/

5. check GA Dashboard and see if the action has been recorded

example screencap:
<img width="990" alt="screen shot 2018-08-13 at 3 09 41 pm" src="https://user-images.githubusercontent.com/2896608/44060876-ef3b343a-9f0a-11e8-8ce2-e496b041e64a.png">
